README.md

```diff
@@ -1,15 +1,89 @@
 # Jubilant, the joyful library for integration-testing Juju charms
 
+![CI Status](https://github.com/canonical/jubilant/actions/workflows/ci.yaml/badge.svg)
+
```

Comment:
It's not important for this PR, but I am curious about your thoughts on badges -- I don't think we've ever talked about those. It feels like badges were super hot for a while (leading to situations like the ops-scenario README until very recently, or even more extreme cases), and are less so now.

These do get used by Jon's releasegen tool (although only the "operator" team repositories show up there, not the "charm tech" team ones), although I feel like the tool probably could get the data without needing the badge, particularly with some consistent naming.

I'm not really convinced of the value of badges, particularly test/build workflow ones. I'm not opposed to them, either (when used judiciously), just not sure what they are bringing. I'm curious about the motivation here.

Reply:
Yeah, this is a good call. I just copied this from Ops without much thought. But I agree with you -- I'm not sure what value it provides, and GitHub already shows a little green tick on the latest commit. Removed.